workers/loc.api/sync/dao/db-migrations/sqlite-migrations/migration.v19.js

```diff
@@ -0,0 +1,353 @@
+'use strict'
+
+const AbstractMigration = require('./abstract.migration')
+
+class MigrationV19 extends AbstractMigration {
+  /**
+   * @override
+   */
+  up () {
+    const sqlArr = [
+      'DROP INDEX ledgers_mts_currency',
+      'DROP INDEX trades_mtsCreate_symbol',
+      'DROP INDEX fundingTrades_mtsCreate_symbol',
+      'DROP INDEX publicTrades_mts__symbol',
+      'DROP INDEX orders_mtsUpdate_symbol',
+      'DROP INDEX movements_mtsUpdated_currency',
+      'DROP INDEX fundingOfferHistory_mtsUpdate_symbol',
+      'DROP INDEX fundingLoanHistory_mtsUpdate_symbol',
+      'DROP INDEX fundingCreditHistory_mtsUpdate_symbol',
+      'DROP INDEX positionsHistory_mtsUpdate_symbol',
+      'DROP INDEX positionsSnapshot_mtsUpdate_symbol',
+      'DROP INDEX logins_time',
+      'DROP INDEX changeLogs_mtsCreate',
+      'DROP INDEX tickersHistory_mtsUpdate_symbol',
+      'DROP INDEX statusMessages_timestamp_key',
+      'DROP INDEX candles_mts__symbol',
+
+      `CREATE INDEX ledgers_user_id_wallet_currency_mts
+        ON ledgers(user_id, wallet, currency, mts)`,
+      `CREATE INDEX ledgers_user_id_wallet_mts
+        ON ledgers(user_id, wallet, mts)`,
+      `CREATE INDEX ledgers_user_id_currency_mts
+        ON ledgers(user_id, currency, mts)`,
+      `CREATE INDEX ledgers_user_id__isMarginFundingPayment_mts
+        ON ledgers(user_id, _isMarginFundingPayment, mts)`,
+      `CREATE INDEX ledgers_user_id__isAffiliateRebate_mts
+        ON ledgers(user_id, _isAffiliateRebate, mts)`,
+      `CREATE INDEX ledgers_user_id__isStakingPayments_mts
+        ON ledgers(user_id, _isStakingPayments, mts)`,
+      `CREATE INDEX ledgers_user_id__category_mts
+        ON ledgers(user_id, _category, mts)`,
+      `CREATE INDEX ledgers_user_id_mts
+        ON ledgers(user_id, mts)`,
+      `CREATE INDEX ledgers_currency_mts
+        ON ledgers(currency, mts)`,
+      `CREATE INDEX ledgers_user_id_subUserId_mts
+        ON ledgers(user_id, subUserId, mts)
+        WHERE subUserId IS NOT NULL`,
+      `CREATE INDEX ledgers_subUserId_mts__id
+        ON ledgers(subUserId, mts, _id)
+        WHERE subUserId IS NOT NULL`,
+
+      `CREATE INDEX trades_user_id_symbol_mtsCreate
+        ON trades(user_id, symbol, mtsCreate)`,
+      `CREATE INDEX trades_user_id_orderID_mtsCreate
+        ON trades(user_id, orderID, mtsCreate)`,
+      `CREATE INDEX trades_user_id_execAmount_mtsCreate
+        ON trades(user_id, execAmount, mtsCreate)`,
```

Comment:
Where do we use this one?

Reply:
It's for UI filters
![Screenshot from 2020-12-01 07-59-31](https://user-images.githubusercontent.com/16489235/100703028-6ad99280-33ab-11eb-8b4f-233f45a17683.png)

Reply:
ok, lets remove this index